worker/provisioner/kvm-broker.go

```diff
@@ -70,7 +70,7 @@ func (broker *kvmBroker) StartInstance(args environs.StartInstanceParams) (*envi
 	if err != nil {
 		// It's not fatal (yet) if we couldn't pre-allocate addresses for the
 		// container.
-		logger.Warningf("failed to prepare container %q network config: %v", machineId, err)
+		logger.Debugf("failed to prepare container %q network config: %v", machineId, err)
```

Comment:
SEt back to info, though I'm hopeful that the new dynamic bridging stuff makes this go away.